### DISCUSSION TITLE
Hello :) when update to vite 5 ?

### DISCUSSION THREAD

**Discussion by ciprianmacovei** [saadeghi/daisyui#2623] [Ideas / feature requests]
require dose not work esm module also, im very interested added the cdn and it works, very nice job, the question in the title remains :)

**Comment by saadeghi** [comment]
Vite is not a dependency of daisyUI. Are you talking about daisyUI document site?  

If there's problem about `require` or ESM, I need a reproduction repo to see the issue. Otherwise it's so hard to guess what is the problem because there are many projects (ESM or CommonJS) using daisyUI and there's no problem.

Let me know if you have a question.

**Comment by ciprianmacovei** [comment]
oooo ur right :))) it works , i was adding it at vite plugins :)))) lol, im ashamed of myself, sorry for that

<img width="552" alt="Screenshot 2023-12-07 at 12 30 52" src="https://github.com/saadeghi/daisyui/assets/23572894/2c3b1686-e5d4-48cb-b25f-c5b3e58a5a32">
